PHYSOR26/common_mesh.i

```diff
@@ -0,0 +1,126 @@
+ref_mesh_file_name=gt.e-s011
+test_mesh_file_name=hex_same_flux_high_err_ma.e-s010
+
+[Mesh]
+  [generated_mesh]
+     type = GeneratedMeshGenerator
+    nx = 20
+    ny = 4
+    nz = 4
+    xmin = 0.0
+    xmax = 100.0
+    ymin = 0.0
+    ymax = 10.0
+    zmin = 0.0
+    zmax = 10.0
+    dim = 3
+  []
+[]
+
+
+[AuxVariables]
+    [adaptive_hierarchy]
+        order = CONSTANT
+        family = MONOMIAL
+    []
+    [element_refinement_step]
+        order = CONSTANT
+        family = MONOMIAL
+    []
+    [test_solution_hierarchy]
+        order = CONSTANT
+        family = MONOMIAL
+    []
+
+    [max_elemental_hierachy]
+        order = CONSTANT
+        family = MONOMIAL
+    []
+    [ref_solution_hierarchy]
+        order = CONSTANT
+        family = MONOMIAL
+    []
+[]
+
+[AuxKernels]
```

Comment:
Here, the most logical order is probably:
* load_ref_solution_hierarcyh
* load_test_solution_hierarchy
* calc_max_element_hierarchy
* adaptive_hierarchy_aux
* calc_remaining_refinement_steps